client/src/js/users/components/__tests__/Users.test.js

```diff
@@ -18,4 +18,79 @@ describe("<ManageUsers />", () => {
         wrapper = shallow(<ManageUsers {...props} />);
         expect(wrapper).toMatchSnapshot();
     });
+
+    it("should render when componentDidMount is called", () => {
+        props.groupsFetched = false;
+        const wrapper = shallow(<ManageUsers {...props} />);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['wrapper' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4529916)